modules/cert-manager-override-arguments.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-arguments_{context}"]
+= Customizing cert-manager Operator API fields by overriding arguments
```

Comment:
Probably, this one could also be re-phrased a bit. How about:
`Customizing cert-manager by overriding arguments from operator API`